SmartDeviceLink/SDLStreamingMediaManager.h

```diff
@@ -118,7 +119,17 @@ NS_ASSUME_NONNULL_BEGIN
  @param configuration The configuration of this streaming media session
  @return A new streaming manager
  */
-- (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connectionManager configuration:(SDLStreamingMediaConfiguration *)configuration NS_DESIGNATED_INITIALIZER;
+- (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connectionManager configuration:(SDLStreamingMediaConfiguration *)configuration __deprecated_msg("Use initWithConnectionManager:streamingMediaConfiguration:lifecycleConfiguration: instead");
+
+/**
+ Create a new streaming media manager for navigation and VPM apps with a specified configuration
+
+ @param connectionManager The pass-through for RPCs
+ @param streamingMediaConfiguration The configuration of this streaming media session
+ @param lifecycleConfiguration The lifecycle configuration of the app
+ @return A new streaming manager
+ */
+- (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connectionManager streamingMediaConfiguration:(SDLStreamingMediaConfiguration *)streamingMediaConfiguration lifecycleConfiguration:(SDLLifecycleConfiguration *)lifecycleConfiguration NS_DESIGNATED_INITIALIZER;
```

Comment:
What about just passing the `SDLConfiguration`?

Reply:
Fixed